graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/aggregations/SupportedGraphqlFunction.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.schema.cqlfirst.dml.fetchers.aggregations;
+
+public enum SupportedGraphqlFunction {
+  INT_FUNCTION("_int_function"),
```

Comment:
Do we have to specify a function for each type? We can't have something more generic and infer the type from the column?

Reply:
yes, we need it. See this discussion for more details:
https://github.com/stargate/stargate/issues/130#issuecomment-825999565